content/source/docs/cloud/workspaces/run-tasks.html.md

```diff
@@ -5,93 +5,91 @@ page_title: "Run Tasks - Workspaces - Terraform Cloud and Terraform Enterprise"
-   - **Mandatory** tasks can block a run from completing. If the task fails (including a timeout or unexpected remote error condition), a warning will be displayed on the run and the run will transition to an Errored state.
-
-
-![Screenshot: enforcement configuration for a specific task within a workspace](./images/run-tasks-add-to-workspace.png)
-
-Your run tasks are now configured. You can return to this page to configure or remove tasks in your workspace.
-
-![Screenshot: final configuration of tasks within a workspace](./images/run-tasks-final-workspace-configuration.png)
+Your run tasks are now configured. You can return to the settings page to configure or remove tasks in your workspace.
+    ![Screenshot: final configuration of tasks within a workspace](./images/run-tasks-final-workspace-configuration.png)
 
 
 ## Understanding Run Tasks Within a Run
 
-Workspaces with run tasks configured have an additional run stage called [Pre-Apply](../run/states.html). This stage is where your run tasks will execute during a run, after the plan stage and also the cost estimation and policy check stages, if applicable.
+Run tasks execute between the [plan](../run/states.html#2-the-plan-stage) and [apply](../run/states.html#6-the-apply-stage) stage of a [Terraform run](../run/index.html). Once all run tasks executions have completed, the run will end based on the most restrictive enforcement level that has been configured in each associated run tasks.
 
-The Pre-Apply stage will always end with the most restrictive status of the tasks configured to run. For example, if a mandatory task fails and an advisory task succeeds, the stage will fail and the run will error. If an advisory task fails but a mandatory task succeeds, the stage will succeed and the run will proceed to the apply stage.
+For example, if a mandatory task fails and an advisory task succeeds, the run will fail. If an advisory task fails, but a mandatory task succeeds, the run will succeed and proceed to the apply stage. Regardless of the exit status of a task, the status and any related message data will be displayed in the UI.
 
-Regardless of the exit status of a task, the status and any related message will be displayed in the UI.
-
-Here is an example of a run that failed due to a mandatory task.
+Here is an example of a run that failed due to a mandatory run task.
 
 ![Screenshot: a run with failed tasks](./images/run-tasks-run-failed.png)
 
 And here is an example of a run that succeeded.
 
-![Screenshot: a run with failed tasks](./images/run-tasks-run-success.png)
+![Screenshot: a run with passed tasks](./images/run-tasks-run-success.png)
+
+## Removing a Run Task from a Workspace
+
+1. Navigate to the desired workspace, then click **Run Tasks** from the **Settings** menu.
 
-## Deleting a Task from a Workspace
+2. Click the ellipses (...) on the associated run task, and then click **Remove**.
```

Comment:
It's just nice to give users one sentence that confirms what they just did and lets them know the result. 

```suggestion
2. Click the ellipses (...) on the associated run task, and then click **Remove**. The run task will no longer be applied to runs within the workspace.
```